Kubernetes-Manifests-file/Database/secrets.yaml

```
@@ -5,5 +5,5 @@ metadata:
   name: mongo-sec
 type: Opaque
 data:  
-  password: cGFzc3dvcmQxMjM=   #Three-Tier-Project
-  username: YWRtaW4= #admin
\ No newline at end of file
+  password: cGFzc3dvcmQxMjM=   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)